src/test/groovy/com/ofg/uptodate/finder/jcenter/JCenterNewVersionFinderFactoryWrongContentTypeSpec.groovy

```diff
@@ -0,0 +1,34 @@
+package com.ofg.uptodate.finder.jcenter
+
+import com.ofg.uptodate.finder.NewFinderSpec
+
+import static com.ofg.uptodate.Xmls.HIBERNATE_CORE_META_DATA
+import static com.ofg.uptodate.Xmls.JUNIT_META_DATA
+
+@Mixin([JCenterResponseProvider])
+class JCenterNewVersionFinderFactoryWrongContentTypeSpec extends NewFinderSpec {
+
+    def setup() {
+        project.extensions.uptodate.jCenterRepo = "http://localhost:${MOCK_HTTP_SERVER_PORT}/"
+        project.extensions.uptodate.ignoreMavenCentral = true
+    }
+
+    @Override
+    protected void artifactMetadataRequestResponse(String group, String name, String response) {
+        stubInteractionForJcenter(group, name, response, "application/unknown")
+    }
+
+    def "should list all dependencies, that have newer versions"() {
```

Comment:
how about including some information about content type in method name (or in `artifactMetadataRequestResponse` method name)?